release/build.gradle

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'groovy'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile 'org.codehaus.groovy:groovy-all:2.4.13'
+    compile 'commons-cli:commons-cli:1.2'
+}
+
+// Nightly releases
+def verNightly = project.findProperty('ver') ?: '2.4.0-SNAPSHOT'
```

Comment:
Applying `build_rules.gradle` would give you the current version at head since that is the authoritative place to get information about the project.

Even though at this moment it is technically incorrect, if you can fix it in this PR that would be great.

Reply:
Fixed by adding the createJavaQuickstartValidationTask function to build_rules.gradle and using the version defined there.